results-processor/main.py

```diff
@@ -202,13 +202,23 @@ def task_handler():
     finally:
         shutil.rmtree(tempdir)
 
+    # Authenticate as "_processor" for create-test-run API.
     ds = datastore.Client()
     secret = ds.get(ds.key('Uploader', '_processor'))['Password']
     test_run_id = wptreport.create_test_run(report, labels, uploader, secret,
                                             results_gcs_path,
                                             raw_results_gcs_path)
     assert test_run_id
 
+    # Authenticate as "_spanner" for push-to-spanner API.
+    secret = ds.get(ds.key('Uploader', '_spanner'))['Password']
+    response = requests.post(
+        config.project_baseurl() + '/api/spanner_push_run',
+        auth=('_processor', secret),
+        data=json.dumps({'run_id': test_run_id}})
+    response.raise_for_status()
```

Comment:
We don't want to raise here (at least for now). We want to return 201 even if this fails; otherwise TaskQueue will retry the task and end up creating duplicate runs in Datastore.

`app.logger.info` any error here.